netlify.toml

```diff
@@ -4,3 +4,7 @@
   base = "code/"
   publish = "build/"
   command = "npm run build"
+  [[redirects]]
+  from = "/*"
+  to = "/index.html"
+  status = 200
```

Comment:
Great job with displaying your own error page instead of the boring Netlify one!